parser/src/ignore_block.rs

```diff
@@ -63,6 +64,27 @@ impl IgnoreBlocks {
     }
 }
 
+pub fn replace_all_and_ignores<'h, R: Replacer>(
```

Comment:
Could you please rename the function? I don't know what `and_ignores` means, tbh :sweat_smile: And also add a short doc comment please.

(Sorry, wanted to post this comment in the first review, but then I changed the comment to "please rewrite the function" xD).

Reply:
If you have a better name I'm all hears.

The only I can come up with that is descriptive enough is `replace_all_with_ignore_blocks_taken_into_account`.

Reply:
`replace_all_outside_ignores` or `replace_all_outside_ignore_blocks`?

Reply:
When with `replace_all_outside_ignore_blocks`.